common/src/main/java/com/wynntils/features/combat/RangeVisualizerFeature.java

```diff
@@ -120,11 +120,11 @@ private void checkMajorIdCircles(Player player) {
         // Offset the radius slightly so multiple circles can be shown for each player
         // Only a few major IDs can actually be applied at the same time, but we make this general
         List<Pair<CustomColor, Float>> circles = validGear.stream()
-                .flatMap(gearInfo -> gearInfo.fixedStats().majorIds().stream().map(majorId -> switch (majorId.id()) {
-                    case "TAUNT" -> Pair.of(CommonColors.ORANGE.withAlpha(TRANSPARENCY), 12f);
-                    case "HERO" -> Pair.of(CommonColors.WHITE.withAlpha(TRANSPARENCY), 8f);
-                    case "ALTRUISM" -> Pair.of(CommonColors.PINK.withAlpha(TRANSPARENCY), 8.1f);
-                    case "GUARDIAN" -> Pair.of(CommonColors.RED.withAlpha(TRANSPARENCY), 7.9f);
+                .flatMap(gearInfo -> gearInfo.fixedStats().majorIds().stream().map(majorId -> switch (majorId.name()) {
```

Comment:
Have Wynncraft dropped the concept of an "id" from a major id, as opposed to a name? Some of these have been renamed in the past, so that sounds like a bad idea from their side.